lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
+ *     the response-content-disposition parameter of the signed url is set 
+ *     accordingly.
+ * @param {string=} options.responseType - If you provide this value, the 
+ *     response-content-type parameter of the signed url is set accordingly.
  * @param {function=} callback - The callback function.
  *
  * @example
  * file.getSignedUrl({
  *   action: 'read',
- *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14) // 2 weeks.
+ *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14), // 2 weeks.
+ *   responseDisposition: 'attachment; filename="filename.ext"'
```

Comment:
Thanks for this PR! I **really** like the idea of making the common case ("make this a download prompt") obvious and simple. Could we possibly provide both?

When we open that door, the question becomes what happens when you specify both fields... Some options are...

1. `responseDisposition` wins (as it's the more specific of the two)
1. `responseDisposition` wins and logs a warning about `promptSaveAs` being ignored
1. We raise an exception saying "Sorry you can't do that"